job/aggregation.go

```diff
@@ -53,8 +61,18 @@ func (a *Aggregation) UnmarshalJSON(b []byte) error {
 		return errors.New("Aggregation limit must be greater than 0")
 	}
 
+	proxy, ok := tmp["aggr_proxy"].(string)
+	if !ok {
+		proxy = ""
+	} else {
+		if _, err := url.ParseRequestURI(proxy); proxy != "" && err != nil {
```

Comment:
I agree, in that scope a `NewJob` could be also added.

Reply:
So how are we going to handle this?

I see two options:

    Create an HTTP client per Job instead of aggregation and set the timeout there
    Add a HUGE default timeout.

Do you have any other (more elegant) ideas?